README.md

```diff
@@ -184,14 +250,13 @@ COMMANDS
 ```
 
 To show the host status:
-
 ```bash
 $ mn status:host
 ```
 
 ### Development
 
-When developing on a standalone node (the `local` preset), `setup-for-local-development` can be used
+When developing on a standalone node (the `local` config), `setup-for-local-development` can be used
```

Comment:
Basically, you can have my-own-dev config. Matter is network option (should be `local`)